### ISSUE TITLE
[BUG] `npm login` with `--scope` not working as expected

### ISSUE THREAD

**Issue by yeskunall** [npm/cli#4098]
### Is there an existing issue for this?

- [X] I have searched the existing issues
> This issue was previously reported in #3292, and fixed in #3383 but that was on a previous release of the CLI.

### This issue exists in the latest npm version

- [X] I am using the latest npm

### Current Behavior

The user `.nvmrc` is created as:

```ini
//registry.npmjs.org/:_authToken = (protected)
```

### Expected Behavior

The `.nvmrc` _should_ look like:

```ini
//scope:registry.npmjs.org/:_authToken = (protected)
```

... where `scope` is the scope defined when doing `npm login` (see below)

### Steps To Reproduce

1. On macOS 11.0.1
2. With the config as seen below in the output of `npm config ls`
3. Run `npm login --scope=<scope>`
4. See output of `npm config ls` (especially the ”user” section)

### Environment

- npm: `v8.1.4`
- Node: `v16.13.0`
- OS: Darwin
- platform: MBP (Intel) running macOS 11.0.1
- npm config:
```ini
; "global" config from /Users/kimchi/.config/npm/.npmrc

fund = false
init.author.email = "hey@kimchiii.space"
init.author.name = "Kunall Banerjee <hey@kimchiii.space> (https://kimchiii.space/)"
init.author.url = "https://kimchiii.space/"
init.license = "MIT"
init.version = "0.0.1"

; "user" config from /Users/kimchi/.config/npm/user/.npmrc

//registry.npmjs.org/:_authToken = (protected)

; "env" config from environment

cache = "/Users/kimchi/.cache/npm"
userconfig = "/Users/kimchi/.config/npm/user/.npmrc"

; node bin location = /Users/kimchi/.config/nvm/versions/node/v16.13.0/bin/node
; cwd = /Users/kimchi/dev/node/site
; HOME = /Users/kimchi
; Run `npm config ls -l` to show all defaults.
```

**Comment by yeskunall** [comment]
This seems to have been fixed in `v8.13.2`.

```sh
; "global" config from /Users/kimchi/.config/npm/.npmrc

fund = false
init-author-email = "hey@kimchiii.space"
init-author-name = "Kunall Banerjee <hey@kimchiii.space> (https://kimchiii.space/)"
init-author-url = "https://kimchiii.space/"
init-license = "MIT"
init-version = "0.0.1"

; "user" config from /Users/kimchi/.config/npm/user/.npmrc

@21kb:registry = "https://registry.npmjs.org/"
//registry.npmjs.org/:_authToken = (protected)

; "env" config from environment

cache = "/Users/kimchi/.cache/npm"
userconfig = "/Users/kimchi/.config/npm/user/.npmrc"
```

Peep the “user” section this time around. HUH. I think I’m going to keep it open till such time a core member can confirm what fixed the bug/regression.

**Comment by wraithgar** [comment]
Yep! This was fixed shortly after you opened this issue https://github.com/npm/cli/pull/4060
